packages/esm-ward-app/src/beds/occupied-bed.component.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { type Patient } from '@openmrs/esm-framework';
+import { type Bed } from '../types';
+import styles from './occupied-bed.scss';
+import { Tag } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import WardPatientCard from '../ward-patient-card/ward-pateint-card.component';
+
+export interface OccupiedBedProps {
+  patients: Patient[];
+  bed: Bed;
+}
+const OccupiedBed: React.FC<OccupiedBedProps> = ({ patients, bed }) => {
+  return (
+    <div className={styles.occupiedBed}>
+      {patients.map((patient, index: number) => {
+        const last = index === patients.length - 1;
+        return (
+          <div key={patient.uuid + ' ' + index}>
+            <WardPatientCard patient={patient} bed={bed} status={'admitted'} />
```

Comment:
An occupied bed != admitted, it just means the bed is occupied.  Admission is a separate, orthongonal concern.